src/config/constants/ifo.ts

```diff
@@ -1,12 +1,9 @@
+import { Token, ChainId } from '@pancakeswap/sdk'
 import tokens from './tokens'
 import farms from './farms'
-import { Ifo, Token } from './types'
+import { Ifo } from './types'
 
-const cakeBnbLpToken: Token = {
-  symbol: farms[1].lpSymbol,
-  address: farms[1].lpAddresses,
-  decimals: 18,
-}
+const cakeBnbLpToken = new Token(ChainId.MAINNET, farms[1].lpAddresses[ChainId.MAINNET], 18, farms[1].lpSymbol)
```

Comment:
It should not change anytime soon but should we start putting the decimals number in a constant to reuse it across the app?

Reply:
We actually do have it in `config/index.ts`, `DEFAULT_TOKEN_DECIMAL`

Reply:
Yea decent point - I guess in this case, `18` is actually a bit easier. One day we'll tell you about `BIG_NINE` @0xCorgi :D